wandb/sdk/internal/internal_api.py

```diff
@@ -3698,6 +3703,11 @@ def create_artifact(
             )
             # ttlDurationSeconds is only usable if ttlIsInherited is also present
             ttl_duration_seconds = None
+        if "tags" not in artifact_fields and tags:
+            wandb.termwarn(
+                "Server not compatible with setting Artifact tags, please upgrade the server to use Artifact tags"
+            )
```

Comment:
please break the line to fit in 80 characters

```suggestion
            wandb.termwarn(
                "Server not compatible with setting Artifact tags,"
                " please upgrade the server to use Artifact tags"
            )
```

also, is this a user-facing message? could we double check with anyone on the wording? should it end with a period?

Reply:
> please break the line to fit in 80 characters

can do -- we might want to consider enforcing this via linter

(for context: I tend to keep IDE settings at 120 chars per line unless required otherwise by ruff/black/etc)

> also, is this a user-facing message? could we double check with anyone on the wording? should it end with a period?

Wording is mainly written for consistency with the existing message -- under same circumstances -- about Artifact TTL: https://github.com/wandb/wandb/blob/f8e60357ef5661b461ead9a8e150e65a589dcf59/wandb/sdk/artifacts/artifact.py#L1072-L1074

Though yeah, it'd make sense to end the message (and the "Artifact TTL" message) with a period